_posts/2018-02-19-Hive-Talks-Security.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Hive Talks - Security
+excerpt: Join the meetup we're organizing at Apiary's Prague office on 6th of March
+layout: post
+date: 2018-02-19 10:00:00 +0200
+author: daria
+published: true
+---
+
+## Cryptography, Blockchain and Machine Learning
+
+Those three fields will come together on the computer security focused meetup oganized as part of our Hive Talk series on 6th of March, 2018.
+
+![Github Repos Association](/images/2018-02-19-Hive-Talks-Security/2018-02-19-Hive-Talks-Logo.png)
+
+Join us, grab a drink and enjoy the talks:
+
+- Petr Svenda ([Enigma Bridge](https://enigmabridge.com/), [Centre for Research on Cryptography and Security](https://crocs.fi.muni.cz/) at Masaryk University) is one of the Czech researchers credited with discovering [vulnerabilities in chips used in Estonias IDs - among others](https://arstechnica.com/information-technology/2017/10/crypto-failure-cripples-millions-of-high-security-keys-750k-estonian-ids/). Petr will talk about the troubles with RSA cryptographic keypair generation.
+
+- Dite Gashi ([Cryptelo](https://www.cryptelo.com/), [Decissio](https://decissio.com/), [Bethereum](https://www.bethereum.com/)) will talk about blockchain powered trust-less data storing and processing. One of Dite's projects, Cryptelo, raised $1 Million USD in a single night of its ICO pre-sale.
```

Comment:
Should `storing` be `storage`?

I think `blockchain powered` should become `blockchain-powered`.

---

> One of Dite's projects, Cryptelo, raised $1 Million USD in a single night of its ICO pre-sale.

I think this should be `during` not `in` to make the grammar of this sentence work.

```diff
- Million USD in a single night of its ICO pre-sale
+ Million USD during a single night of its ICO pre-sale
```